### PR TITLE
Re-add Copay v3.9.1

### PR DESCRIPTION
**Pull request by FranklinYu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
This was removed by https://github.com/caskroom/homebrew-cask/pull/28405 in favour of `bitpay`.

**Comment by commitay** [comment]
@franklinyu Can you please explain the difference between the two and why this version is needed?

**Comment by franklinyu** [comment]
I think the app was renamed last year, but now it branched into two applications. Nevertheless I think @johnrees may know more than I do.

**Comment by franklinyu** [comment]
From the GitHub release history, on 25 Oct 2016 they decide to release BitPay. According to [official blog](https://blog.bitpay.com/introducing-bitpay-app/), they didn't intend to rename, but to release a new application instead. We can see from the next release on 22 Mar this year (3.0.6), both Copay and BitPay is available.

**Comment by commitay** [comment]
> ... both Copay and BitPay is available

This their any difference between the two apart from branding? i.e What is the reason for re-adding `copay`?

**Comment by franklinyu** [comment]
According to the blog I mentioned and [this FAQ](https://support.bitpay.com/hc/en-us/articles/115003028366-How-is-the-BitPay-wallet-app-related-to-Copay-How-is-it-different-), only BitPay works with the BitPay Visa, which we can just use as an ordinary Visa credit card. In addition, the blog seems to imply that BitPay is *not* free-software, and Copay is the "community version".

File: Casks/copay.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'copay' do
+  version '3.9.1'
+  sha256 '1300dd42e801c7e82754a601fb9e1953242e41f25e29ab4d9bf43376ff288ec6'
+
+  # github.com/bitpay/copay was verified as official when first introduced to the cask
+  url 'https://github.com/bitpay/copay/releases/download/v#{version}/Copay.dmg'
```
Comment (commitay) [inline Casks/copay.rb]: This will fail `style`, it should be double quoted.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/copay.rb (hunk 1 above)
Comment (franklinyu) [reply Casks/copay.rb]: My bad, didn't re-check style after amend.

Comment (commitay) [reply Casks/copay.rb]: Thanks!

**Review by commitay — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Sounds like a kind of Chromium/Chrome situation.

**Pull request merged**